build/yaml/crd/vpc/crd.nsx.vmware.com_ipaddressallocations.yaml

```diff
@@ -3,7 +3,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.11.0
```

Comment:
It looks that the version revert is not expected?

Reply:
ok